lib/msal-common/src/logger/Logger.ts

```diff
@@ -62,10 +62,16 @@ export class Logger {
         const defaultLoggerCallback = () => {
             return;
         };
-        this.localCallback = loggerOptions.loggerCallback || defaultLoggerCallback;
-        this.piiLoggingEnabled = loggerOptions.piiLoggingEnabled || false;
-        this.level = typeof(loggerOptions.logLevel) === "number" ? loggerOptions.logLevel : LogLevel.Info;
-        this.correlationId = loggerOptions.correlationId || Constants.EMPTY_STRING;
+        const defaultLoggerOptions = {};
+        if(!loggerOptions)
+        {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
This lint rule is important, please refactor rather than ignoring

Reply:
Have refactored to use a default setter method